react/ImageList.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { Fragment } from 'react'
+import { defineMessages } from 'react-intl'
+import { useDevice } from 'vtex.device-detector'
+
+import Image from './Image'
+
+interface Props {
+  images: {
+    link?: {
+      url: string
+      noFollow: boolean
+      openNewTab: boolean
+      title: string
+    }
+    image: string
+    mobileImage: string
+    description: string
+  }[]
+  height: number
+}
+
+const ImageList: StorefrontFunctionComponent<Props> = ({
+  images,
+  height = 420,
+}) => {
+  const { isMobile } = useDevice()
+
+  return (
+    <Fragment>
+      {images.map(({ link, image, mobileImage, description }, idx) => (
+        <Image
+          key={idx}
+          src={isMobile && mobileImage ? mobileImage : image}
+          alt={description}
+          link={link}
+          maxHeight={height}
+        />
+      ))}
+    </Fragment>
+  )
+}
+
+const messages = defineMessages({
+  title: { id: 'admin/editor.image-list.title', defaultMessage: '' },
+  description: {
+    id: 'admin/editor.image-list.description',
+    defaultMessage: '',
+  },
+  imagesImageTitle: {
+    id: 'admin/editor.image-list.images.image.title',
+    defaultMessage: '',
+  },
+  imagesMobileImageTitle: {
+    id: 'admin/editor.image-list.images.mobileImage.title',
+    defaultMessage: '',
+  },
+  imagesImageDescription: {
+    id: 'admin/editor.image-list.images.description.title',
+    defaultMessage: '',
+  },
+  imagesImageLinkUrl: {
+    id: 'admin/editor.image-list.images.link.url.title',
+    defaultMessage: '',
+  },
+  imagesImageLinkOpenNewTab: {
+    id: 'admin/editor.image-list.images.link.openNewTab.title',
+    defaultMessage: '',
+  },
+  imagesImageLinkNoFollow: {
+    id: 'admin/editor.image-list.images.link.noFollow.title',
+    defaultMessage: '',
+  },
+  imagesImageLinkTitle: {
+    id: 'admin/editor.image-list.images.link.title.title',
+    defaultMessage: '',
+  },
+  imagesTitle: {
+    id: 'admin/editor.image-list.images.title',
+    defaultMessage: '',
+  },
+  heightTitle: {
+    id: 'admin/editor.image-list.height.title',
+    defaultMessage: '',
+  },
+})
+
+ImageList.schema = {
+  title: messages.title.id,
+  description: messages.description.id,
+  type: 'object',
+  properties: {
+    images: {
+      items: {
+        properties: {
+          image: {
+            default: '',
+            title: messages.imagesImageTitle.id,
```

Comment:
You should check with the site-editor team, but I believe this should be in the contentSchema

Reply:
Yes! You're right, but I deleted this component in the next Pull Request in favor of `ImageSlider`, that one uses the `contentSchema` ;)